package.json

```diff
@@ -1353,8 +1353,7 @@
                     "default": [],
                     "items": {
                         "enum": [
-                            "CustomEditor",
-                            "All"
+                            "CustomEditor"
```

Comment:
Was it just to remove the autocomplete for this? experiements/services.ts still has the code in it to use 'All' it just would not show up as an autocomplete option.

Reply:
Looks good. At least that's assuming that we want to remove the option from people who have already specified it? But that was how I understood it at least.